modules/ROOT/pages/index.adoc

```diff
@@ -5,6 +5,14 @@ endif::[]
 
 This topic describes the Anypoint Platform Private Cloud Edition. Anypoint Platform Private Cloud Edition enables you to run and manage Mule applications on your local servers, using the security policies your system requires. When running Anypoint Platform Private Cloud Edition, MuleSoft systems do not interact with your data. You configure and run the software and maintain all storage, processing and transmission of data.
 
+The Anypoint Platform Private Cloud Edition provides the following application components when combined with CloudHub:
+
+* Runtime Manager: enables you to track and manage your deployed Mule applications
```

Comment:
Kim likes a newline after the :

Reply:
Suggest no : but make each of these full sentences with terminating punctuation.
Runtime Manager enables you to track and manage your deployed Mule applications.

Reply:
Thank you - I updated the wording.